docs/requirements.txt

```diff
@@ -0,0 +1,3 @@
+mkdocs
+mkdocs-material
+mkdocstrings[python]
```

Comment:
I'd prefer not to duplicate this with our `pyproject.toml`, but I'm not sure there's any way to get RTD to use those. I'll play around with that once we get RTD running.

Reply:
RTD looks pretty inflexible with how it handles requirements, so I think the only other option would be to make a script to generate this file on-the-fly from our `pyproject.toml`, which sounds like more trouble and complexity than it's probably worth.